apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -494,17 +494,17 @@ def grade(element_html, data):
 def parse_correct_answer(f, a):
     if f == "homogeneous":
         try:
-            T = np.array(a, dtype=np.float64)
-            if T.shape == (4, 4):
-                R = T[0:3, 0:3]
-                p = T[0:3, 3:4]
-                return np.reshape(p, (3,)), pyquaternion.Quaternion(matrix=R)
+            transform_matrix = np.array(a, dtype=np.float64)
+            if transform_matrix.shape == (4, 4):
+                rot_matrix = transform_matrix[0:3, 0:3]
```

Comment:
Nit: `rotation_matrix` or `rotate_matrix`? Trying to wean us off of using overly terse names for things in Python 😅 ditto elsewhere in the file if you decide to change this.